test/jaxp/javax/xml/jaxp/unittest/catalog/CatalogTest.java

```diff
@@ -506,7 +506,7 @@ public void testRewriteUri() throws Exception {
     */
     @Test(expectedExceptions = NullPointerException.class)
     public void testFeatureNull() {
-        CatalogResolver resolver = CatalogManager.catalogResolver(null, null);
+        CatalogResolver resolver = CatalogManager.catalogResolver((CatalogFeatures)null, (URI)null);
```

Comment:
I misunderstood this change, as I thought this was for testing `null` for the *new* method. Should we have the equivalent one for the new method?

Reply:
True, added a test. Thanks.